src/Microsoft.Diagnostics.Monitoring.EventPipe/Configuration/MetricSourceConfiguration.cs

```diff
@@ -30,8 +30,10 @@ public sealed class MetricEventPipeProvider
 
     public sealed class MetricSourceConfiguration : MonitoringSourceConfiguration
     {
+        public const string SharedSessionId = "SHARED";
```

Comment:
`dotnet monitor` and `dotnet counters` both use a special-case session id as previously discussed (SHARED)